### DISCUSSION TITLE
Cycle to restore original size/position

### DISCUSSION THREAD

**Discussion by Stooovie** [rxhanson/Rectangle#653] [Ideas]
I propose a setting that cycling a shortcut restores the original size and position of the window (for example, if a window is already on the left half, a repeated command to size to left half would restore the window). IIRC that's Aero Snap on Windows works.

Thanks.

**Comment by rxhanson** [comment]
Makes sense, and shouldn't be hard to add in. I'll take a look for next release.

**Reply by shakedbr** [reply]
Hi, are you still planning to implement this feature?

**Reply by rxhanson** [reply]
At the moment, I don't have the bandwidth. I'll merge a good pull request for it.

**Comment by LPFchan** [comment]
![SCR-20230426-pxi](https://user-images.githubusercontent.com/1547081/234537449-409ed36d-0372-477a-9fc9-7717de29a7bb.png)

Hi, Is this feature implemented in some sort of way? I wish double-activating the maximize shortcut would restore the window to its initial size & position ...
